mne/minimum_norm/spatial_resolution.py

```diff
@@ -65,19 +65,14 @@ def resolution_metrics(resmat, src, function='psf', metric='peak_err',
 
     Notes
     -----
-    For details, see [1]_ [2]_.
+    For details, see :footcite:`MolinsEtAl2008` :footcite:`HaukEtAl2019`.
 
     .. versionadded:: 0.20
-
+    
```

Comment:
you added some trailing whitespace here: That's undesirable (invisible, useless characters) --> a good solution would be to look into your editor settings. For most editors you can have an automatic removal of trailing whitespace :-)